docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,680 @@
+
+. (Optional) Verify that the instance has been created in the table on the *Kafka Instances* page of the {product-kafka} {service-url-kafka}[web console^].
+. To connect your applications or services to a Kafka instance in {product-kafka}, you must first create a service account with credentials. Copy and paste the example `rhoas_service_account` resource into the `main.tf` file to create a service account. This example uses the `"my-service-account"` identifier and creates a service account called my-service-account.
++
+
+.Example `rhoas_service_account` resource
+[source]
+----
+resource "rhoas_service_account" "my-service-account" {
+  name        = "my-service-account"
+  description = "<description of service account>"
+}
+
+output "client_id" {
+  value = rhoas_service_account.my-service-account.client_id
+}
+
+output "client_secret" {
+  value     = rhoas_service_account.my-service-account.client_secret
+  sensitive = true
+}
+----
++
+. Save your changes.
+. Apply the changes you made to your Terraform provider configuration.
++
+[source, shell]
+----
+$ terraform apply
+----
+In the terminal, Terraform displays a message that rhoas_service_account.my-instance will be created.
+. When you're ready to create your service account, type *yes*. The generated client ID appears in the terminal as an output. The client secret does not display because it is marked as a sensitive value.
+. (Optional) Verify that the service account has been created in the table on the *Service Accounts* page of the {product-kafka} {service-url-kafka}[web console^].
+. Copy and paste the example `rhoas_topic` resource into the `main.tf` file to create a Kafka topic with default values. This example uses the `topic` identifier and creates the `"my-topic"` Kafka topic. As you have already created the Kafka instance, Terraform can check dependencies for this new topic resource and knows the Kafka ID when you run this example resource.
++
+.Example `rhoas_topic` resource with default values
+[source]
+----
+resource "rhoas_topic" "topic" {
+		name = "my-topic"
+		partitions = 1
+		kafka_id = rhoas_kafka.instance.id
+	}
+
+----
++
+. Save your changes.
+. Apply the changes you made to your Terraform provider configuration.
++
+[source, shell]
+----
+$ terraform apply
+----
+In the terminal, Terraform displays a message that `rhoas_topic.my-topic` will be created.
+. When you're ready to create your topic, type *yes*.
+. (Optional) Verify that the topic is listed on the *Topics* page of the {product-kafka} {service-url-kafka}[web console^].
+. Copy and paste the `rhoas_acl` resource into the `main.tf` file to create an ACL for the service account with some default values, and bind that ACL to the Kafka instance and service account. This example uses the `"acl"` identifier. The following list describes each field in an ACL binding resource.
```

Comment:
"To create Access Control List (ACL) permissions for the service account with some default values, copy..."